### DISCUSSION TITLE
postgresql field type inet

### DISCUSSION THREAD

**Discussion by rotkaepchen** [dbeaver/dbeaver#11731] [Q&A]
Hello,
I have a postgresql table with a column type INET.
As driver I use postgres-jdbc 42.2.19.
If I try to update a field I get a failure:
>Object parameter [1.1.1.1/24] binding not supported

What did I have to configure to update the fields?

Thanks :-)

**Comment by LonwoLonwo** [comment]
Hello @rotkaepchen 
And what version of DBeaver do you use?
"Object parameter binding" problem was in previous releases, but we fixed it.

**Comment by rotkaepchen** [comment]
Hello @LonwoLonwo,
I use the version 7.2.4 from 11/2020.
With the current version 21.0 the failure is fixed.
Thanks for the hint
